PukingPredator/Assets/Scripts/GameInput.cs

```diff
@@ -10,6 +10,7 @@ public class GameInput : MonoBehaviour
     private PlayerInput playerInput;
     public EventHandler onInteractAction;
     public EventHandler onPukeAction;
+    public EventHandler onResetAction;
```

Comment:
would this be on level reset? Could we rename it to `onLevelResetAction` please?